DEVNOTES.md

```diff
@@ -16,21 +16,22 @@ Install [`poetry`](https://python-poetry.org/), then install the project:
 ~/FastqHeat$ poetry install  # NOTE: includes dev dependencies
 ```
 
-> **NOTE**: to run commands within the project's virtual environment you will have
-> to activate Poetry's shell (`poetry shell`) or run them via `poetry run`. If that
-> sounds too complicated, install the project via `pip` in editable mode (`-e`)
-> instead, and then install project's dependencies with `poetry install --no-root`.
-
```

Comment:
This text was removed but where is this instruction now?

Reply:
> Then we need examples and right commands to do.

I did add examples that use `poetry run`. They are one paragraph below.

> The second solution provides **another** way to run commands

Yes, but as I explained, it's an inferior way to install and launch this project for development purposes.